lib/compat.php

```diff
@@ -366,11 +366,18 @@ function gutenberg_replace_default_block_categories( $default_categories ) {
 }
 add_filter( 'block_categories', 'gutenberg_replace_default_block_categories' );
 
+global $current_parsed_block;
+$current_parsed_block = array(
+	'blockName'  => null,
+	'attributes' => null,
+);
+
 /**
  * Shim that hooks into `pre_render_block` so as to override `render_block` with
  * a function that assigns block context.
  *
- * This can be removed when plugin support requires WordPress 5.5.0+.
+ * FIXME: THIS IS NO LONGER ACCURATE: This can be removed when plugin support
```

Comment:
If we keep `$current_parsed_block`, then this needs to be updated to reflect that:

1. It needs backporting to core
2. Then, it needs to live in compat.php until 5.6 is required by the plugin

Reply:
Hm, good catch. Is this filtering out any elements in `$extra_attributes` that aren't `style` or `class`? I think we should fix that. Even if keep that restriction for `$new_attributes` (which makes sense) I think we should lift it for `$extra_attributes`, otherwise it IMO goes against developer expectations.